### PR TITLE
SKALE-3158 Add healthcheck for skale-admin

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: healthchecks/admin.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os
+import time
+
+MAX_ALLOWED_LOG_TIME_DIFF = os.environ.get('MAX_ALLOWED_LOG_TIME_DIFF', 600)
```
Comment (badrogger) [inline healthchecks/admin.py]: os.environ.get -> os.getenv

File: healthchecks/admin.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+modification_time = os.path.getmtime(ADMIN_LOG_FILEPATH)
+current_time = time.time()
+time_diff = current_time - modification_time
+
+print(f'Modification time diff: {time_diff}')
+if time_diff > MAX_ALLOWED_LOG_TIME_DIFF:
+    exit(3)
+else:
+    exit(0)
```
Comment (badrogger) [inline healthchecks/admin.py]: I think we should add if __name__ == '__main__'

**Review by badrogger — CHANGES_REQUESTED** [review]

File: healthchecks/admin.py (hunk 1 above)
Comment (dmytrotkk) [reply healthchecks/admin.py]: done

File: healthchecks/admin.py (hunk 2 above)
Comment (dmytrotkk) [reply healthchecks/admin.py]: done

**Review by badrogger — DISMISSED** [review]
Great!

**Pull request merged**